demo/stop.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ -f "demo.pid" ]
+then
+    kill -9 $(cat demo.pid)
+    rm demo.pid
+fi
```

Comment:
I'd refactor to make all of this a more future-ready example, where this might be one of a few, therefore the name of each file should reflect an ease of adding more vs this being singular. Aside from that, looks good.